SYCL/Basic/barrier_order.cpp

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED: hip
```

Comment:
hip or hip_amd?

Reply:
hip, this is the same as my other comment, it's an issue in the plugin, it will also fail on nvidia